### PR TITLE
issue #31: test running without API key and test cases for query_creator

### PR DESCRIPTION
**Pull request by anoopvalluthadam**

### CODE REVIEW

**Comment by anoopvalluthadam** [comment]
@sylvestre Please do let me know whether the fix is fine.

File: auto_nag/bugzilla/utils.py [hunk 1]
```diff
@@ -6,6 +6,23 @@
 import urllib
 
 
+def get_project_root_path():
```
Comment (sylvestre) [inline auto_nag/bugzilla/utils.py]: Would be great to have a test for this new method

File: auto_nag/bugzilla/utils.py [hunk 2]
```diff
@@ -6,6 +6,23 @@
...
+    """
+    Get project root path
+    return:string: Project ROOT folder
+    """
+    pwd = os.getcwd()
+    return pwd.split('auto_nag')[0] + '/'
+
+
+def get_config_path():
```
Comment (sylvestre) [inline auto_nag/bugzilla/utils.py]: Would be great to have a test for this new method

File: auto_nag/tests/test_query_creator.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        """
+        Expecting a VERSION number
+        """
+        url = "https://wiki.mozilla.org/Template:BETA_VERSION"
+        beta_version = getTemplateValue(url)
+        assert type(int(beta_version)) is not type(int)
+
+    def test_getReportURL(self):
+        """
+        Expectong proper URL with Bug numbers
```
Comment (sylvestre) [inline auto_nag/tests/test_query_creator.py]: expecting I guess ;)

Comment (anoopvalluthadam) [inline auto_nag/tests/test_query_creator.py]: Anu is writing test cases for utils, so she told me she will write today
itself if my pull got merged

On Mon, Oct 26, 2015, 1:04 PM Sylvestre Ledru <notifications@github.com>
wrote:

> In auto_nag/tests/test_query_creator.py
> <https://github.com/mozilla/relman-auto-nag/pull/38#discussion_r42965391>:
>
> > +    sys.path.insert(0, parentdir)
> > +    from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
> > +                                                createQueriesList)
> > +
> > +class TestQueryCreator:
> > +    def test_getTemplateValue(self):
> > +        """
> > +        Expecting a VERSION number
> > +        """
> > +        url = "https://wiki.mozilla.org/Template:BETA_VERSION"
> > +        beta_version = getTemplateValue(url)
> > +        assert type(int(beta_version)) is not type(int)
> > +
> > +    def test_getReportURL(self):
> > +        """
> > +        Expectong proper URL with Bug numbers
>
> expecting I guess ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/relman-auto-nag/pull/38/files#r42965391>.
>
-- 

~anoop_valluthadam

Comment (anoopvalluthadam) [inline auto_nag/tests/test_query_creator.py]: @sylvestre do yo want me to remove that?

Comment (sylvestre) [inline auto_nag/tests/test_query_creator.py]: no, I was just taking about the typo (expectong instead of expecting)

Comment (anoopvalluthadam) [inline auto_nag/tests/test_query_creator.py]: OMG, I didn't see that ;) 

thanks!

**Pull request merged**